modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -4,18 +4,18 @@
 
 [id="dedicated-cluster-install-deploy"]
 
-= Installing OpenShift Logging and Elasticsearch Operators
+= Installing OpenShift Logging and OpenShift Elasticsearch Operators
 
 You can use the {product-title} console to install OpenShift Logging by deploying instances of
-the OpenShift Logging and Elasticsearch Operators.  The Cluster Logging Operator
-creates and manages the components of the logging stack.  The Elasticsearch Operator
+the OpenShift Logging and OpenShift Elasticsearch Operators.  The Cluster Logging Operator
```

Comment:
Maybe replace the instances of `.  ` (two spaces) here with `. ` (one space)?

Reply:
Fixed two in this file.